data/mods/Magiclysm/enchanted/enchanted_belts.json

```diff
@@ -0,0 +1,186 @@
+[
+  {
+    "abstract": "mbelt_leather",
+    "type": "TOOL_ARMOR",
+    "name": "magic leather belt",
+    "description": "A leather belt.  Useful for making your pair of pants fit.",
+    "weight": "102 g",
+    "volume": "250 ml",
+    "price": 90000,
+    "bashing": 5,
+    "material": [ "leather" ],
+    "symbol": "[",
+    "color": "brown",
+    "covers": [ "TORSO" ],
+    "coverage": 5,
+    "material_thickness": 2,
+    "use_action": {
+      "type": "holster",
+      "holster_prompt": "Stick what into your belt",
+      "holster_msg": "You tuck your %s into your %s",
+      "max_volume": "500 ml",
+      "max_weight": 400,
+      "draw_cost": 60,
+      "flags": [ "BELT_CLIP" ]
+    },
+    "flags": [ "WAIST", "WATER_FRIENDLY", "STURDY" ]
+  },
+  {
+    "type": "TOOL_ARMOR",
+    "copy-from": "mbelt_leather",
+    "id": "mbelt_haste",
+    "name": "Belt of Haste",
+    "relic_data": {
+      "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "SPEED", "multiply": 0.15, "add": 5 } ] } ]
+    }
+  },
+  {
+    "type": "TOOL_ARMOR",
+    "copy-from": "mbelt_leather",
+    "id": "mbelt_thor",
+    "name": "Megingjörð",
+    "description": "The mythical belt of Thor, god of thunder.  Or at least so it appears.  It doubles the wearer's base strength.",
+    "material": [ "superalloy" ],
```

Comment:
i'm not so sure about it being superalloy. maybe when that mythological metals and weapons pr gets in we can switch it over to orichalcum.

Reply:
I'm not using orichalcum for anything norse, that's a greek mythological material, basically brass.
This was basically done to make it indestructible, whenever something else presents itself as an option I'd go with that.